hr_timesheet_sheet_view_adj/views/hr_timesheet_sheet_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_hr_timesheet_sheet_filter" model="ir.ui.view">
+        <field name="name">hr_timesheet_sheet.sheet.filter</field>
+        <field name="model">hr_timesheet_sheet.sheet</field>
+        <field name="inherit_id"
+               ref="hr_timesheet_sheet.view_hr_timesheet_sheet_filter"/>
+        <field name="arch" type="xml">
+            <xpath expr="filter[@name='message_needaction']" position="before">
+                <separator />
+                <filter 
+                    name="subject_to_approve" 
+                    string="Subject to approve" 
```

Comment:
I think the string is a bit misleading since another filter handles the 'to approve' part. I would set it as "My Team Members" or something like that.

Reply:
@yostashiro チームミーティングの時にTimさんと、「チーム」だと必ずしも同じチームでないこともあるかもしれないということで、「承認対象」にしよう、と話したのですが、"My Team Members"の方がいいでしょうか。
「承認対象」の英訳を"Subject to approve"としたのですが、確かに"to approve"のフィルターと名前がかぶるので変えたいですね。

Reply:
> 「チーム」だと必ずしも同じチームでないこともあるかもしれない

@Ryoko04 How about just 'My Members', then?